### ISSUE TITLE
Android Support

### ISSUE THREAD

**Issue by ravitez** [rauchy/react-native-offline-mode#5]
Works perfectly with IOS.
Any plan to support on Android ?

**Comment by JackThomson2** [comment]
It works on Android for me

**Comment by ravitez** [comment]
Yes,expected to work on Android as well as it's a js only library. Any specific configuration needed  for Android ?  I followed the usual setup and added permission for network access on Android*.xml but no luck..

**Comment by JackThomson2** [comment]
Hi, it has been a while since I used this component but I think you need this permission: android.permission.ACCESS_NETWORK_STATE

Thanks Jack

**Comment by grozen** [comment]
I suspect this is an actual issue that will go away with the merging of #7.

**Comment by NemSothea** [comment]
change from: let connect = (reach) => this.setState({ isConnected: reach !== 'none' })
to: let connect = (reach) => this.setState({ isConnected: reach.toLowerCase() !== 'none' })
because on Android when there is no network connection it will produce 'NONE', but on iOS will be 'none'.
so here is the difference and the problem was solved.
